docs/spec/api.md

```diff
@@ -1108,7 +1133,9 @@ A list of methods and URIs are covered in the table below:
 |Method|Path|Entity|Description|
 |------|----|------|-----------|
 | GET | `/v2/` | Base | Check that the endpoint implements Docker Registry API V2. |
-| GET | `/v2/<name>/tags/list` | Tags | Fetch the tags under the repository identified by `name`. |
+| GET | `/v2/<name>/tags/_list` | Tags | Fetch the tags under the repository identified by `name`. |
+| GET | `/v2/<name>/tags/<reference>` | Tags | Redirect to tags list. |
```

Comment:
Sorry, just reread your comment.  If only `list` is supported as `<reference>` why bother to call it `reference`?